core-bundle/contao/templates/modules/mod_two_factor.html5

```diff
@@ -20,6 +20,8 @@
         </div>
         <div class="widget">
           <p><?= $this->trans('MSC.twoFactorTextCode') ?></p>
+          <?php $this->addCspHash('style-src', 'word-break:break-all'); ?>
+          <?php $this->addCspSource('style-src', 'unsafe-hashes'); ?>
```

Comment:
Can't we automate this? It's always needed when adding a hash for inline styles, isn't it?

Reply:
Only for inline style _attributes_, not for inline style _elements_.

Reply:
To elaborate on this here as well - see https://content-security-policy.com/examples/allow-inline-style/

> CSP Level 3 (newest browsers) support a source list value: [`unsafe-hashes`](https://content-security-policy.com/unsafe-hashes/) which can be used to allow inline style attributes on HTML tags.

The `unsafe-hashes` source is only required for `style="…"` HTML attributes, not for `<style>` tags. Since I think we should not automatically allow any directive sources that are not necessary in general, I think we should not add `unsafe-hashes` automatically either when using `addHash`/`add_csp_hash`.